service/src/test/java/uk/gov/hmcts/reform/fpl/service/ApplicantServiceTest.java

```diff
@@ -130,4 +135,26 @@ void shouldNotAddNewPartyIdWhenApplicantsAlreadyHasPartyIdValue() {
 
         assertThat(applicant.getParty().partyId).isEqualTo(uuid);
     }
+
+    @Test
+    void shouldReturnApplicantCollectionWithOrganisationDetailsWhenOrganisationExists() {
+        CaseData caseData = CaseData.builder().build();
+
+        List<Element<Applicant>> applicants = service.expandApplicantCollection(caseData, buildOrganisation());
+        assertThat(applicants.get(0).getValue().getParty().getOrganisationName())
+            .isEqualTo(buildOrganisation().getName());
+    }
+
+    @Test
+    void shouldReturnApplicantCollectionWithoutOrganisationDetailsWhenNoOrganisationExists() {
+        CaseData caseData = CaseData.builder().build();
+
+        List<Element<Applicant>> applicants = service.expandApplicantCollection(caseData,
+            Organisation.builder().build());
+        assertThat(applicants.get(0).getValue().getParty().getOrganisationName()).isNull();
+    }
+
+    private Organisation buildOrganisation() {
+        return Organisation.builder().name("Organisation").build();
```

Comment:
could we not set this as a constant and use everywhere. No difference between using this and using Organisation.builder().build()

Reply:
do you mean like this at top of class

 private final Organisation ORGANISATION = Organisation.builder().name("Organisation").build();

instead of a method?

Reply:
Actually i've changed this buildOrganisation method to include the address now also so i don't think this comment is relevant anymore

Reply:
changed